### DISCUSSION TITLE
[Improvment}: Latest UI update

### DISCUSSION THREAD

**Discussion by Woisek** [vladmandic/sdnext#1218] [Ideas]
I noted from the latest update, that "the five icons" we knew for so long under Stop and Skip, are changed. As they are now hard to recognize and even more optically away from their function they represent, may I suggest to at least add a title attribute with the function name to them, so that we know what the do when hovering over them?
It would be generally a good practice to "titeling" all buttons so that even new users can see what the are/do.

Thanks!

**Comment by vladmandic** [comment]
i'll bring hints back, they are hidden inadvertently
